torch/_subclasses/functional_tensor.py

```diff
@@ -149,11 +150,18 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
             def unwrap(x):
                 return x.elem
 
-            assert len(args) == 1 and isinstance(args[0], FunctionalTensor)
-            assert len(kwargs) == 0
             # All metadata accesses should be plumbed to the inner tensor, that way we don't have to worry
             # about the problem of keeping metadata in sync between the wrapper and inner tensor.
             # This also alleviates us from having to manually handle metadata mutations on the wrapper.
+            assert len(kwargs) == 0
+            if func in [
+                torch.ops.aten.is_strides_like_format.default,
+                torch.ops.aten.is_contiguous.memory_format,
+            ]:
+                assert len(args) == 2 and isinstance(args[0], FunctionalTensor)
+                return func(args[0].elem, args[1])
+            assert len(args) == 1 and isinstance(args[0], FunctionalTensor)
```

Comment:
Btw, what's going on here?

Reply:
I think the previous assumption of all metadata query functions only take one argument was bit buggy. For above two metadata query functions, it takes two arguments

Reply:
oof. I'm curious how you hit this in the PR though 🤔